### DISCUSSION TITLE
Task suggestions for database team

### DISCUSSION THREAD

**Discussion by YampierFernandez** [uprm-inso4101-2024-2025-s1/semester-project-mercado-colegial-online#31] [Ideas]
-Research host - ends when host is chosen – 2 members

-Research language - ends when language is chosen - 2 members

-Research extensions - ends when extensions are chosen - 2 members

-Resources for learning - ends when every member post at least 1 resource - 2 members

-Proof of set up - ends when every member provides evidence of their set up - all members of the team

**Comment by franrov4** [comment]
Hello Team,

It’s crucial that all members set up their development environments as soon as possible. We will be assigning a high-priority task for this, as it is essential for the next phase of our project. Please prioritize this setup.


Additionally, I encourage everyone to suggest new tasks using the template below:

	•	Objective:
	•	Requirements:
	•	Completion Criteria:
	•	Time Constraint:

	•	Urgency:
	•	Difficulty:

The sooner we generate these tasks, the more efficiently we can move forward and ensure success for both the project and individual team members. 

Note: Timely task creation will also positively impact overall performance scores.

Thank you!

**Comment by JuanIranzo** [comment]
Since Node.JS is already uploaded to the repo, the MongoDB setup is almost done, and we already have a ERD to kind of guide us, could we already start implementing the logic for the tables, controllers for the entities, etc. this week? If so, this could open up a bunch of tasks for the whole team

**Reply by YampierFernandez** [reply]
Yes, for this week the basic setup is done so we will focus on task for populating the database and defining the logic. David mentioned in another discussion that the next step should be setting up the first cluster, so that for example is a task we could create.
